src/mngt/ocf_mngt_cache.c

```diff
@@ -1539,6 +1539,207 @@ struct ocf_pipeline_properties _ocf_mngt_cache_attach_pipeline_properties = {
 	},
 };
 
+typedef void (*_ocf_mngt_cache_unplug_end_t)(void *context, int error);
```

Comment:
Please modify commit message to make it clear, that stop actually can return error code. However the only possible non-zero status is just a warning and cache is always stopped after executing ocf_mngt_cache_stop().